tests/actions.go

```diff
@@ -507,12 +507,12 @@ func (oi *OperatorConfig) Enabled(feature string) bool {
 
 func (oa *operatorActions) runKubectlOrDie(args ...string) string {
 	cmd := "kubectl"
-	klog.Infof("Running '%s %s'", cmd, strings.Join(args, " "))
+	k8se2elog.Logf("Running '%s %s'", cmd, strings.Join(args, " "))
 	out, err := exec.Command(cmd, args...).CombinedOutput()
 	if err != nil {
-		klog.Fatalf("Failed to run '%s %s'\nCombined output: %q\nError: %v", cmd, strings.Join(args, " "), string(out), err)
+		k8se2elog.Failf("Failed to run '%s %s'\nCombined output: %q\nError: %v", cmd, strings.Join(args, " "), string(out), err)
```

Comment:
Will this abort as `Fatalf`?

Reply:
Yes. This will eventually call `ginkgo.Fail` and there's a `panic` at the end.